conanfile.py

```diff
@@ -25,7 +25,7 @@ def source(self):
         os.rename("libxml2-{0}".format(self.version), self.source_subfolder)
 
     def config_options(self):
-        if self.settings.os == "Windows":
+        if self.settings.os == "Windows" and self.settings.compiler == "Visual Studio":
```

Comment:
fPIC is not needed for MinGW as well, leave just `if self.settings.os == Windows`